src/FileExtensionManager.h

```diff
@@ -0,0 +1,30 @@
+#ifndef FILEEXTENSIONMANAGER_H
+#define FILEEXTENSIONMANAGER_H
+
+#include <QDialog>
+
+namespace Ui {
+class FileExtensionManager;
+}
+
+class FileExtensionManager : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit FileExtensionManager(QStringList init, QWidget *parent = 0);
```

Comment:
Can you change that ```0``` to ```nullptr```? Just for consistency with the rest of the code :smile:

Reply:
Of course